CHANGELOG.md

```diff
@@ -11,6 +11,10 @@
 - (Splunk) Add Azure Blob receiver ([#5200](https://github.com/signalfx/splunk-otel-collector/pull/5200))
 - (Splunk) Add Google Cloud PubSub receiver ([#5200](https://github.com/signalfx/splunk-otel-collector/pull/5200))
 
+### 💡 Enhancements 💡
+
+- (Splunk) Upgrade golang to 1.22 ([#5248](https://github.com/signalfx/splunk-otel-collector/pull/5248))
```

Comment:
I believe this is a breaking change (that's what we've labeled it historically)

Reply:
> I believe this is a breaking change (that's what we've labeled it historically)

Moved to breaking changes.